tests/cargo-ui/function-stubbing-trait-mismatch/src/expected

```diff
@@ -0,0 +1,3 @@
+error: Unable to resolve call to `DoIt::do_it`; this is either an internal Kani bug, or there is a trait mismatch between a stub you are using and the original function/method.
```

Comment:
I had two tests because one treats the code as a library (the `ui` one) and one treats it as an executable (the `cargo-ui` one). These end up hitting the validation code along different code paths. Is there a better way to test this? Also, what sort of sanity check for `cargo-ui` did you have in mind?